docs/reference/authentication.md

```diff
@@ -27,6 +27,17 @@ Web UI at Administration / Users to create users.  Users have an access key
 `AKIA...` and an associated secret access key.  These credentials are valid
 to log into the Web UI, or to authenticate programmatic requests to the API
 Server or the S3 Gateway.
+git
+#### lakeFS Invite
+
+If you were invited to lakeFS, Follow these steps to create and activate your account:
+
+1. Locate the email sent to you, (be aware it may be in your spam folder)
+1. Click on the link in the email, to be directed to the account activation page
```

Comment:
The second half of the sentence is in the passive voice.  See [style basics](https://docs.google.com/document/d/1vif05nDiKZYPwYdINLoq4BJ6j7bBIQ6ZA2dt7McuzyY/edit#heading=h.dqp77semtfge).

```suggestion
1. Click on the link in the email to open the account activation page.
```

Reply:
Both are active voices in my opinion, but I do accept the change